bench-run/targets.mk

```diff
@@ -0,0 +1,53 @@
+# Use host network at dockerfiles builds
```

Comment:
Why do you add docker stuff if your patch is aimed at not using docker? Or I am missing something?

Reply:
I don't use docker now only for sysbench. Until all benches will not use docker we have to leave docker settings and files.

Reply:
Got it.